### ISSUE TITLE
[Bug] ERROR    Error in _handle_one_transaction, closing:

### ISSUE THREAD

**Issue by neo57500** [Chia-Network/chia-blockchain#14236]
### What happened?

2022-12-27T21:28:55.537 full_node chia.full_node.full_node: ERROR    Error in _handle_one_transaction, closing: Traceback (most recent call last):
  File "chia/full_node/full_node.py", line 458, in _handle_one_transaction
  File "chia/full_node/full_node.py", line 2262, in respond_transaction
AssertionError

2022-12-27T21:28:55.538 full_node chia.full_node.full_node: ERROR    Error in _handle_one_transaction, closing: Traceback (most recent call last):
  File "chia/full_node/full_node.py", line 458, in _handle_one_transaction
  File "chia/full_node/full_node.py", line 2262, in respond_transaction
AssertionError

2022-12-27T21:28:55.555 full_node chia.full_node.full_node: ERROR    Error in _handle_one_transaction, closing: Traceback (most recent call last):
  File "chia/full_node/full_node.py", line 458, in _handle_one_transaction
  File "chia/full_node/full_node.py", line 2262, in respond_transaction
AssertionError

2022-12-27T21:28:56.117 full_node full_node_server        : ERROR    Exception: Failed to fetch block 3023567 from PeerInfo(host='78.139.80.139', port=8444), timed out, PeerInfo(host='78.139.80.139', port=8444). Traceback (most recent call last):
  File "chia/server/ws_connection.py", line 376, in wrapped_coroutine
  File "chia/full_node/full_node_api.py", line 133, in new_peak
  File "chia/full_node/full_node.py", line 737, in new_peak
  File "chia/full_node/full_node.py", line 655, in short_sync_backtrack
ValueError: Failed to fetch block 3023567 from PeerInfo(host='78.139.80.139', port=8444), timed out

### Version

1.7.0b4

### What platform are you using?

Linux

### What ui mode are you using?

GUI

### Relevant log output

```shell
2022-12-27T21:28:55.537 full_node chia.full_node.full_node: ERROR    Error in _handle_one_transaction, closing: Traceback (most recent call last):
  File "chia/full_node/full_node.py", line 458, in _handle_one_transaction
  File "chia/full_node/full_node.py", line 2262, in respond_transaction
AssertionError
```

**Comment by MumfMeisterT** [comment]
We are working on this. Here is a PR if you would like to try it. If so, please let us know your results. 
https://github.com/Chia-Network/chia-blockchain/pull/14274

**Comment by emlowe** [comment]
We believe this is fixed by #14274 which is in 1.7.0 release - closing as Fixed
